text_adventure.py

```diff
@@ -0,0 +1,190 @@
+import random
+
+class GameObject:
+    verb_dict = {
+        "examine": "Examine a creature in the room.",
+        "hit": "Hit a creature in the room.",
+        "help": "Get help about the available commands.",
+        "heal": "Heal for an amount of damage.",
+        "kill": "Immediately kill the target.",
+        "status": "Get your own status."
+    }
+
+    @staticmethod
+    def get_input():
+        command = input(": ").split()
+        verb_word = command[0]
```

Comment:
this is great! what you've done here is created a grammar, look up ABNF, lex, and parse as keywords for further study.